tests/acceptance/bs-navbar-test.js

```diff
@@ -0,0 +1,13 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+// TODO: Write this
+moduleForAcceptance('Acceptance | bs navbar');
+
+test('visiting /navbars', function(assert) {
+  visit('/navbars');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/navbars');
+  });
+});
```

Comment:
We should reconsider this if we decide to back off on the contextual components. The navbar test won't be nearly as comprehensive on the multi-component behaviors in that case.